### DISCUSSION TITLE
non-variable d in dx

### DISCUSSION THREAD

**Discussion by hamaney** [typst/typst#4165] [Q&A] [answered]
#The Goal
<img width="262" alt="image" src="https://github.com/typst/typst/assets/11434409/8ec9fc8f-7998-4e35-9e19-a9973cbbba8f">

# What I have

<img width="226" alt="image" src="https://github.com/typst/typst/assets/11434409/d05be813-e3c8-4a5f-8ef9-0e69921271e4">

How to make the d to be non-variable?

I have this   $ "{" a,b,c, d/e, integral x^3"d"x "}" &= "ugly" $ but it does not seem to give the effect I wanted

**Comment by Enivex** [comment, marked as answer]
`upright(d)` or `dif` (also adds space)

**Reply by hamaney** [reply]
Nice. Thank you so much!
